server/container_create_linux.go

```diff
@@ -676,6 +676,30 @@ func (s *Server) createSandboxContainer(ctx context.Context, containerID, contai
 		specgen.AddAnnotation(fmt.Sprintf("%s.%d", annotations.IP, idx), ip)
 	}
 
+	// systemd expects to have /run, /run/lock and /tmp on tmpfs
+	// It also expects to be able to write to /sys/fs/cgroup/systemd and /var/log/journal
+	if s.config.CgroupManager == oci.SystemdCgroupsManager {
```

Comment:
This shouldn't be done unconditionally; key off the command to know that user is intending to runc systemd.

Reply:
fixed